lhotse/dataset/collation.py

```diff
@@ -1,11 +1,109 @@
-from typing import Iterable, Union
+from typing import Iterable, Union, Literal, Tuple, Callable
+from dataclasses import dataclass
 
 import numpy as np
 import torch
 from torch.nn import CrossEntropyLoss
 
 from lhotse import CutSet
 from lhotse.cut import MixedCut
+from lhotse.utils import LOG_EPSILON
+
+
+@dataclass
+class CollateTokens:
```

Comment:
style - either it's mimicking a function and then we should name it `collate_tokens`, or it's an object and then we should name it `TokenCollater` - I am leaning towards the latter

same for other classes of this type

Reply:
Agree, lets use `TokenCollater`